packages/api/src/interfaces.ts

```diff
@@ -3,6 +3,7 @@ import {Resolves} from "./utils/types.js";
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
+export type ResponseFormat = "json" | "ssz";
```

Comment:
This type is also used by the debug state routes.  Should we put this somewhere that both can pull from the same type?

https://github.com/ChainSafe/lodestar/blob/97dfcfc45c098e4a1363bb0213695916eeb94487/packages/api/src/beacon/routes/debug.ts#L25